doc/manual/installation/installing-binary.xml

```diff
@@ -135,113 +147,267 @@ sudo rm /Library/LaunchDaemons/org.nixos.nix-daemon.plist
   </para>
 
 </section>
-
-<section xml:id="sect-apfs-volume-installation">
-  <title>APFS Volume Installation</title>
+<!-- TODO: delete when done-ish
+  Sketching my outline goal
+  installing on macOS
+    recommended/official process
+    overview and/or guide to other options
+      symlink
+      encrypted
+        mount
+        wait4path
+        login hook script
+        keychain
+        plaintext
+    Notes
+      synthetic.conf
+      firmlink
+      t2?
+-->
+
+<section xml:id="sect-macos-installation">
+  <title>macOS Installation</title>
 
   <para>
-    The root filesystem is read-only as of macOS 10.15 Catalina, all writable
-    paths were moved to a separate data volume.  This means creating or writing
-    to <filename>/nix</filename> is not allowed.  While changing the default prefix
-    would be possible, it's a very intrusive change that has side effects we want to
-    avoid for now.
+    Starting with macOS 10.15 (Catalina), the root filesystem is read-only.
+    This means <filename>/nix</filename> can no longer live on your system
+    volume, and that you'll need a workaround to install Nix.
   </para>
 
   <para>
-    For common writable locations <literal>firmlinks</literal> were introduced,
-    described by Apple as a "bi-directional wormhole" between two filesystems.
-    Essentially a bind mount for APFS volumes.  However this is (currently) not
-    user configurable and only available for paths like <filename>/Users</filename>.
+    The recommended approach, which creates an unencrypted APFS volume
+    for your Nix store and a "synthetic" empty directory to mount it
+    over at <filename>/nix</filename>, is least likely to impair Nix
+    or your system.
```

Comment:
Given that all of the current options have drawbacks I wonder if we should list the ideal solutions here for reference.

- firmlink, similar solution as the symlink but without any impact on builds
- filevault, use the same role as the data volume instead of a new passphrase described a bit here https://mjtsai.com/blog/2019/06/06/backing-up-macos-10-15-beta

Reply:
I've pushed a commit that attempts to address this with a new note.